data/roles.js

```diff
@@ -852,81 +878,85 @@ const roleData = {
         "Amnesiac": {
             alignment: "Independent",
             description: [
-                "Once per game can become the role of a dead player."
+                "Choose to become the role of a dead player, once per game."
```

Comment:
Consistency. Should be "Chooses" instead of "Choose." I also don't personally think it needs a comma, but that's just me.